avocado/core/runner.py

```diff
@@ -176,6 +177,11 @@ def _tick(self):
             elif "paused" in msg:
                 self.status = msg
                 self.job.result_proxy.notify_progress(False)
+                # TODO: reuse the job result_events_dispatcher, that
+                # is, either keep it private and create a property to
+                # get it from the test
+                result_events_dispatcher = dispatcher.ResultEventsDispatcher()
+                result_events_dispatcher.map_method('test_progress', False)
```

Comment:
This is what I had been looking for in https://github.com/avocado-framework/avocado/pull/1583/commits/a3f74f44ba54f33c883b009123ec7939b5c2ccbf I think it should me squashed, because the plugin does nothing without this dispatcher.